contrib/roles/linux/kubernetes/tasks/main.yml

```diff
@@ -43,15 +43,14 @@
 
 - name: Kubernetes | Get Kubernetes binaries
   include_tasks: ./download_k8s_binaries.yml
-  when: binaries_missing or master
+  when: binaries_missing
 
 - name: Kubernetes | Fetch Kubernetes binaries to the ansible host
   include_tasks: ./fetch_k8s_binaries.yml
   when: binaries_missing
 
 - name: Kubernetes | Distribute binaries for minions
```

Comment:
If you're now distributing binaries to both master & minions you should change the task name as well.